src/components/UtilizationBar/UtilizationBar.js

```diff
@@ -0,0 +1,94 @@
+import { ProgressBar } from '../ProgressBar';
+import PropTypes from 'prop-types';
+import { Tooltip } from '../Tooltip';
+import { OverlayTrigger } from '../OverlayTrigger';
+import React from 'react';
+
+const availableTooltip = props => {
+  return (
+    <Tooltip id="availableTooltip">Available {props.max - props.now} %</Tooltip>
+  );
+};
+const usedTooltip = props => {
+  return <Tooltip id="usedTooltip">Used {props.now} %</Tooltip>;
+};
```

Comment:
Can you move those components (`availableTooltip` and `usedTooltip`) to
`src/components/UtilizationBar/InnerComponents/AvailableTooltip.js`
`src/components/UtilizationBar/InnerComponents/UsedTooltip.js`

as described here:
https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md

Reply:
If I do that it doesn't work when I call it as a component `<UsedTooltip now={now}>` but it has to be called as `UsedTooltip(now)`.